.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: ci
+
+on:
+  # TODO: these should be tightened up
+  # push:
+  #   branches: "*"
+  pull_request:
+    branches: "*"
```

Comment:
As discussed in the 2021-03-24 meeting, we should have a dedicated branch (not `master`/`main`) on which to gate this. Some ideas:
- `work`
- `wip`
- `ci`

Reply:
I would go with `ci` for now as a branch name.

Reply:
did so: my personal fork now has a `ci` branch, synced to _this_ repos HEAD, so should be [right as :cloud_with_rain: ](https://github.com/bollwyvl/accessibility/pull/1/checks?check_run_id=2193242624).